docs/wcs/examples/from_file.py

```diff
@@ -20,8 +20,11 @@ def load_wcs_from_file(filename):
     w.wcs.print_contents()
 
     # Three pixel coordinates of interest.
-    # Note we've silently assumed a NAXIS=2 image here
-    pixcrd = np.array([[0, 0], [24, 38], [45, 98]], np.float_)
+    # Note we've silently assumed a NAXIS=2 image here.
+    # Note also that the pixel coordinates are pairs of [X, Y]
+    # and since WCS built from FITS header automatically has
+    # origin set to 1, [0, 0] is actually not inside the image.
```

Comment:
The meaning of origin has confused many.
Is it clearer to say something like:
```
Note also that the pixel coordinates are pairs of [X, Y]
and that "origin" is set to the default value of 1 because 
the FITS standard assumes the first pixel in an image has
 coordinates (1,1). This means that the first pixel we pass, (0, 0)
is not in the image. To tell astropy.wcs that these are zero-based coordinates,
we can pass origin=0.

```

Reply:
Thanks, @nden!

@kakirastern , can you please apply the suggested verbiage in both places? Thank you for your continued patience!

Reply:
@pllim Yup, am looking into the matter...

Reply:
@pllim Think I have committed the changes to the two files concerned. But do you think it would be better to put the explanation somewhere outside the example so it comes after the code as an aside? Meaning I will add the following: 

> Note also that the pixel coordinates are pairs of [X, Y] and that "origin" is set to the default value of 1 because the FITS standard assumes the first pixel in an image has coordinates (1,1). This means that the first pixel we pass, (0, 0) is not in the image. To tell astropy.wcs that these are zero-based coordinates, we can pass origin=0.

I think this bit of information is of some importance to be treated with care. Not everyone is aware of it.